pkg/lockutil/lockutil_windows.go

```diff
@@ -31,13 +31,12 @@ func WithDirLock(dir string, fn func() error) error {
 	}
 	defer dirFile.Close()
 	// see https://msdn.microsoft.com/en-us/library/windows/desktop/aa365203(v=vs.85).aspx
-	// 1 lock immediately
```

Comment:
This was never "1 = lock immediately". It is "1 = The function returns immediately if it is unable to acquire the requested lock. Otherwise, it waits.". We obviously want "wait".